src/base/bittorrent/session.cpp

```diff
@@ -440,28 +461,36 @@ Session::Session(QObject *parent)
     pack.set_bool(libt::settings_pack::listen_system_port_fallback, false);
     pack.set_str(libt::settings_pack::user_agent, USER_AGENT);
     pack.set_bool(libt::settings_pack::use_dht_as_fallback, false);
+#if LIBTORRENT_VERSION_NUM < 10200
     // Disable support for SSL torrents for now
     pack.set_int(libt::settings_pack::ssl_listen, 0);
     // To prevent ISPs from blocking seeding
     pack.set_bool(libt::settings_pack::lazy_bitfields, true);
+#endif
     // Speed up exit
     pack.set_int(libt::settings_pack::stop_tracker_timeout, 1);
     pack.set_int(libt::settings_pack::auto_scrape_interval, 1200); // 20 minutes
     pack.set_int(libt::settings_pack::auto_scrape_min_interval, 900); // 15 minutes
     pack.set_int(libt::settings_pack::connection_speed, 20); // default is 10
     pack.set_bool(libt::settings_pack::no_connect_privileged_ports, false);
+#if LIBTORRENT_VERSION_NUM < 10200
     // Disk cache pool is rarely tested in libtorrent and doesn't free buffers
     // Soon to be deprecated there
     // More info: https://github.com/arvidn/libtorrent/issues/2251
     pack.set_bool(libt::settings_pack::use_disk_cache_pool, false);
+#endif
     // libtorrent 1.1 enables UPnP & NAT-PMP by default
     // turn them off before `libt::session` ctor to avoid split second effects
     pack.set_bool(libt::settings_pack::enable_upnp, false);
     pack.set_bool(libt::settings_pack::enable_natpmp, false);
     pack.set_bool(libt::settings_pack::upnp_ignore_nonrouters, true);
     configure(pack);
 
+#if LIBTORRENT_VERSION_NUM < 10200
     m_nativeSession = new libt::session(pack, 0);
+#else
+    m_nativeSession = new libt::session(pack, libt::session::add_default_plugins);
```

Comment:
Perhaps I misremember, but the `0` in the top version is also the flags, and passing different flags will give you different behaviours in the different versions